warc_metadata_sidecar.py

```diff
@@ -143,15 +144,15 @@ def create_string_payload(mime_dict, puid, result_dict, lang_cld, soft404):
     """Collect content mime, puid, encoding, and language to create record payload."""
     payload = []
     if mime_dict:
-        payload.append('{0} {1}'.format(MIME_TITLE, mime_dict))
+        payload.append('{0} {1}'.format(MIME_TITLE, json.dumps(mime_dict)))
     if puid:
-        payload.append('{0} {1}'.format(PUID_TITLE, puid))
+        payload.append('{0} {1}'.format(PUID_TITLE, json.dumps(puid)))
     if result_dict.get('encoding'):
-        payload.append('{0} {1}'.format(CHARSET_TITLE, result_dict))
+        payload.append('{0} {1}'.format(CHARSET_TITLE, json.dumps(result_dict)))
     if lang_cld:
-        payload.append('{0} {1}'.format(LANGUAGE_TITLE, lang_cld))
+        payload.append('{0} {1}'.format(LANGUAGE_TITLE, json.dumps(lang_cld)))
     if soft404 is not None:
-        payload.append('{0} {1}'.format(SOFT404_TITLE, soft404))
+        payload.append('{0} {1}'.format(SOFT404_TITLE, json.dumps(soft404)))
     return '\n'.join(payload)
```

Comment:
Noting that after looking back at the request to change values in the payload to serialized JSON, we have determined via Slack conversation that we don't need to do a `json.dumps` on the `puid` nor the `soft404`.